.github/workflows/certora.yml

```diff
@@ -13,20 +13,35 @@ on:
   workflow_dispatch:
 
 jobs:
-  verify_stkBNB:
+  get-diff:
```

Comment:
this whole job can be combined with the diff job in the `ci.yml` workflow. That would be a better optimization.
We won't need a separate `certora.yml` I believe.

Reply:
Done